publish-backend/tests/app.test.js

```diff
@@ -0,0 +1,29 @@
+const fs = require("fs");
+const { setupStrapi, cleanupStrapi } = require("./helpers/strapi");
+const {
+  createTestUsers,
+  createTestTags,
+  createTestPosts,
+} = require("./helpers/data.helper");
+const { testUsers, testTags, testPosts } = require("./helpers/fixtures");
+
+beforeAll(async () => {
+  // Create a Strapi instance in the testing environment
+  await setupStrapi();
+
+  // Seed test database
+  await createTestUsers(testUsers);
+  await createTestTags(testTags);
+  await createTestPosts(testPosts);
+});
+
+afterAll(async () => {
+  await cleanupStrapi();
+});
```

Comment:
```suggestion
afterAll(cleanupStrapi);
```
cleanup strapi is already an async function, there's no need to wrap it.